### DISCUSSION TITLE
formTab组件当type='object'时，切换tab输入不了

### DISCUSSION THREAD

**Discussion by tank80** [alibaba/formily#1941] [Q&A 2.x] [answered]
sandbox如下：
https://codesandbox.io/s/determined-mountain-yor2p?file=/App.tsx

场景：我一个schema里面要渲染两个formTab组件，如果slot1和slot2的type='void'，这个时候因为两个slot里面的属性名称相同，输入会发生联动，所以打算让slot1和slot2的type='object'，这时切换tab发现输入不了

**Comment by janryWang** [comment, marked as answer]
FormTab不能作为数据节点，因为它有onChange属性，每次切换的时候会触发onChange，其实相当于把对象值改成了字符串，数据对不上了

**Reply by tank80** [reply]
那这种场景有解决方案吗，如果用void的话

**Reply by janryWang** [reply]
多套一层object不就解决了

**Reply by tank80** [reply]
好的，感谢
